src/svymean.jl

```diff
@@ -68,9 +68,9 @@ function svymean(x::Vector{Symbol}, design::SimpleRandomSample)
 end
 
 """
-Inner method for `svyby`.
+Inner method for `svyby` to calculate standard error of domain mean of SimpleRandomSample.
```

Comment:
The extra description is not necessary here. We should describe the basic functionality above the main method and for other methods only mention what is special (if it's not obvious). In this case I would keep the old docstring.

Reply:
removed in latest commit